StarVMC/Geometry/StarGeo.xml

```diff
@@ -61,15 +60,15 @@
     <Construct sys="epdm" config="EPDMv1"  />
     <Construct sys="magp" config="MAGPon"  />
     <Construct sys="mutd" config="MUTD16"  />
-    <Construct sys="fstm" config="FSTMv1" />
-    <Construct sys="stgm" config="STGCv1" />
-    <!-- Construct sys="etof" config="ETOFv14f"  -->
     <Construct sys="wcal" config="WCALv1" simu="1" />
     <Construct sys="hcal" config="HCALv1" simu="1" />
-    <Construct sys="plat" config="PLATon" simu="2" />
-    <Construct sys="pres" config="PRESof" simu="2" /> 
+    <!-- Note:  These versions of the forward trackers have significant overlaps which need to be resolved -->
```

Comment:
Not quite sure I understand the overlap comment for FSTMv1. I agree that the STGCv1 is not correct. It is being worked on.